pkg/resourcekeeper/cache.go

```diff
@@ -101,3 +101,13 @@ func (cache *resourceCache) get(ctx context.Context, mr v1beta1.ManagedResource)
 	}
 	return entry
 }
+
+func (cache *resourceCache) setNotExist(mr v1beta1.ManagedResource) {
```

Comment:
I'm a bit confused of what is this function for.

Reply:
When gc by the reversal order of the creation, if the comp's resource is already gc, i want to mark it in the resourceCache.That's why we have this function